### PR TITLE
@damassi => [FEATURE ArtistPage] Update default sort on artist page

### PR DESCRIPTION
**Pull request by mzikherman**
Updates to be merchandisability.

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 1eef9c07a3b60965add1d02c0c9d81733d2e3a95

https://deploy-preview-1411--artsy-reaction.netlify.com

File: src/Apps/Artist/routes.tsx [hunk 1]
```diff
@@ -69,8 +69,13 @@ export const routes: RouteConfig[] = [
...
           // FIXME: The initial render includes `location` in props, but subsequent
           // renders (such as tabbing back to this route in your browser) will not.
-          const initialFilterState = props.location ? props.location.query : {}
-          return { ...initialFilterState, ...params }
+          const filterStateFromUrl = props.location ? props.location.query : {}
+          const filterParams = Object.assign(
+            { ...filterStateFromUrl },
+            { ...params },
+            initialState
+          )
```
Comment (damassi) [inline src/Apps/Artist/routes.tsx]:
```js
const filterParams = {
 ...filterStateFromUrl,
 ...params,
 ...initialState
}
```
Seems weird to mix object.assign and a spread, and also to spread just a single object into itself 

```
{ ...filterStateFromUrl }
```

Comment (mzikherman) [reply src/Apps/Artist/routes.tsx]: yea good call, this is weird

Comment (mzikherman) [reply src/Apps/Artist/routes.tsx]: Updated to the
```
const filterParams = {
 ...filterStateFromUrl,
 ...params,
 ...initialState
}
``` style.

With spreading you don't need `defaults` and `assign`, etc. as I'm used to.

(but, you do need to do it in reverse order from the order you would when using one of the above methods).

Comment (damassi) [reply src/Apps/Artist/routes.tsx]: This is a pretty common pattern with spreads: 

```js
const config = {
  pageNumberDefault: 10,
  someOtherThing: "hi!",
  ...overrides
}
```

**Pull request merged**